beeswax/logs/ad_log.proto

```diff
@@ -14,6 +14,7 @@ syntax = "proto2";
 package logs;
 
 option java_package = "com.beeswax.logs";
+option cc_enable_arenas = true;
```

Comment:
Why are we adding this?

Reply:
It's on the internal version of the file.

Reply:
This seems to be a part of the code generation specs. https://developers.google.com/protocol-buffers/docs/reference/arenas I'm not sure why we have it internally or if we want to have this be the default for external customers.